### PR TITLE
calendar and add patients

### PR DESCRIPTION
**Pull request by sharleenawinja**
# Calendar and add patients

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/RdeSync/AddPatients/AddPatients.tsx
Comment (kipchubett) [inline src/components/RdeSync/AddPatients/AddPatients.tsx:1]: For simplicity and consistency
Add this model/type in the same file since it's not reused

**Review by kipchubett — APPROVED** [review]
Thanks for working on this @sharleenawinja

File: src/components/RdeSync/AddPatients/AddPatients.tsx
Comment (sharleenawinja) [reply src/components/RdeSync/AddPatients/AddPatients.tsx:1]: I'm using it in both the resources and component files, should I create the interface individually in each file?

Comment (kipchubett) [reply src/components/RdeSync/AddPatients/AddPatients.tsx:1]: If that is the case, just rename the file to `model.ts`

**Comment by kipchubett** [comment]
Reworked on PR #57

**Pull request closed without merging**